README.md

```diff
@@ -34,15 +34,24 @@ After compiling a Cairo0 program to `program.json`, run:
 stone-prover-cli prove program.json
 ```
 
-### Run and prove one or more programs/PIEs with the Starknet bootloader
+### Run and prove one or more programs/PIEs
+
+#### Running with the Starknet bootloader
 
 If you want to prove one or more programs and PIEs by running them with the Starknet bootloader,
-you can use the `--with-bootloader` option.
+you can use the `--with-bootloader` option which is enabled by default.
 
 ```shell
 stone-prover-cli prove --with-bootloader program1.json program2.json pie1.zip
 ```
 
+#### Running without the Starknet bootloader
+You can use the `--no-bootloader` option if you want to run without the Starknet bootloader.
+
+```shell
+stone-prover-cli prove --no-bootloader program1.json program2.json pie1.zip
```

Comment:
Missed this, this doesn't work. The VM can only run JSON program files without the bootloader, and only one file at a time. So the call would be `stone-prover-cli prove --no-bootloader program.json`.

Can you update the doc?